src/pages/Swap/AdvancedTradingViewBox/AdvancedSwapMode/AdvancedSwapMode.component.tsx

```diff
@@ -68,6 +69,13 @@ export const AdvancedSwapMode = ({ children }: PropsWithChildren) => {
     ? tradesWrapper?.current?.clientHeight - TradesWrapperHeight.PADDING
     : TradesWrapperHeight.MIN
 
+  const allTokens = useAllTokens()
+
+  console.log({
```

Comment:
maybe we want to remove the logs

Reply:
Addressed in https://github.com/ImpeccableHQ/swapr-dapp/pull/1646/commits/83426e029f16a9fc9ce601d3cecea919e55cd64f